R/osmactive.R

```diff
@@ -52,7 +53,7 @@ exclude_bicycle_cycling = function() {
 # Exclude highway values for driving
 exclude_highway_driving = function() {
   to_exclude = paste0(
-    "crossing|disused|emergency|escap|far|raceway|rest|track",
+    "crossing|services|disused|emergency|escap|far|raceway|rest|road|track",
```

Comment:
What's the rationale behind removing unclassified `highway=road` tags @joeytalbot 

Just had a look at this and see no driving restriction: https://wiki.openstreetmap.org/wiki/Tag:highway%3Droad

Reply:
Looking here, none of these seem to be highways to actually fine with that: https://overpass-turbo.eu/?template=key-value&key=highway&value=road